app/src/pages/lloydGeorgeRecordPage/LloydGeorgeRecordPage.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react';
+import { usePatientDetailsContext } from '../../providers/patientProvider/PatientProvider';
+import { getFormattedDate } from '../../helpers/utils/formatDate';
+import { useNavigate } from 'react-router';
+import { routes } from '../../types/generic/routes';
+import { Card, Details } from 'nhsuk-react-components';
+import { useBaseAPIUrl } from '../../providers/configProvider/ConfigProvider';
+import getLloydGeorgeRecord from '../../helpers/requests/lloydGeorgeSearchResult';
+import PdfViewer from '../../components/generic/pdfViewer/PdfViewer';
+import { getFormattedDatetime } from '../../helpers/utils/formatDatetime';
+import { DOWNLOAD_STAGE } from '../../types/generic/downloadStage';
+import formatFileSize from '../../helpers/utils/formatFileSize';
+
+function LloydGeorgeRecordPage() {
+    const [patientDetails] = usePatientDetailsContext();
+    const [downloadStage, setDownloadStage] = useState(DOWNLOAD_STAGE.INITIAL);
+    const [numberOfFiles, setNumberOfFiles] = useState(0);
+    const [totalFileSizeInByte, setTotalFileSizeInByte] = useState(0);
+    const [lastUpdated, setLastUpdated] = useState('');
+    const [lloydGeorgeUrl, setLloydGeorgeUrl] = useState('');
+    const navigate = useNavigate();
+    const baseUrl = useBaseAPIUrl();
+
+    const dob: String = patientDetails?.birthDate
+        ? getFormattedDate(new Date(patientDetails.birthDate))
+        : '';
+
+    const nhsNumber: String =
+        patientDetails?.nhsNumber.slice(0, 3) +
+        ' ' +
+        patientDetails?.nhsNumber.slice(3, 6) +
+        ' ' +
+        patientDetails?.nhsNumber.slice(6, 10);
+
+    const patientInfo = (
+        <>
+            <p style={{ marginBottom: 5, fontWeight: '700' }}>
+                {`${patientDetails?.givenName} ${patientDetails?.familyName}`}
+            </p>
+            <p style={{ fontSize: '16px', marginBottom: 5 }}>NHS number: {nhsNumber}</p>
+            <p style={{ fontSize: '16px' }}>Date of birth: {dob}</p>
+        </>
+    );
+
+    useEffect(() => {
+        if (!patientDetails) {
+            navigate(routes.HOME);
```

Comment:
This block of code for checking if a patient exists then returning home, no longer necessary on pages as the 'PatientGuard' component on the Router handles this and takes us to 'Unauthorised'

Reply:
Just have everything in the else statement